### PR TITLE
Update sriov_cni

### PR DESCRIPTION
**Pull request by Eoghan1232**

### CODE REVIEW

**Comment by Eoghan1232** [comment]
This should resolve the open CVE's on the sriov_cni pkg

File: sriov_cni/pkg/config/config.go [hunk 1]
```diff
@@ -7,26 +7,39 @@ import (
...
 // LoadConf parses and validates stdin netconf and returns NetConf object
 func LoadConf(bytes []byte) (*sriovtypes.NetConf, error) {
 	n := &sriovtypes.NetConf{}
 	if err := json.Unmarshal(bytes, n); err != nil {
 		return nil, fmt.Errorf("LoadConf(): failed to load netconf: %v", err)
 	}
 
-	if n.DpuDaemonAddress == "" {
-		n.DpuDaemonAddress = addr
+	if n.IpuManagerAddress == "" {
```
Comment (Eoghan1232) [inline sriov_cni/pkg/config/config.go:39]: rename var to DpuDaemonAddress

**Review by ahalimx86 — APPROVED** [review]
Since this changes are mostly from upstream repo, skipping detailed review of the changes assuming all existing changes are merged on top of upstream changes.

**Review by palade — APPROVED** [review]
Approved on the assumption that all existing changes are merged on top of upstream changes.

**Comment by Eoghan1232** [comment]
all changes are present as I did a diff to confirm.
I don't have an env setup to do an e2e test

**Pull request merged**